xml/System.Runtime.InteropServices/SafeBuffer.xml

```diff
@@ -447,9 +447,19 @@ finally {
         <ReturnType>System.Boolean</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
-        <value>To be added.</value>
-        <remarks>To be added.</remarks>
+        <summary>Gets a value that indicates whether the handle is invalid.</summary>
+        <value>
+          <see langword="true" /> if the handle is not valid; otherwise, <see langword="false" />.</value>
+        <remarks>
+          <format type="text/markdown"><![CDATA[
+
+## Remarks
+ The common language runtime uses this property to determine whether critical finalization is required.
+
+ ]]></format>
+        </remarks>
+        <altmember cref="P:System.Runtime.InteropServices.SafeHandleMinusOneIsInvalid.IsInvalid" />
+        <altmember cref="T:System.Runtime.InteropServices.SafeHandleMinusOneIsInvalid" />
```

Comment:
These two lines threw a build warning:

```
2 invalid cross reference(s)
"<xref:System.Runtime.InteropServices.SafeHandleMinusOneIsInvalid.IsInvalid>",
"<xref:System.Runtime.InteropServices.SafeHandleMinusOneIsInvalid>".
```
I didn't find those two APIs inside the `System.Runtime.InteropServices` namespace. They live inside `Microsoft.Win32`:

```suggestion
        <altmember cref="P:Microsoft.Win32.SafeHandles.SafeHandleMinusOneIsInvalid.IsInvalid" />
        <altmember cref="T:Microsoft.Win32.SafeHandles.SafeHandleMinusOneIsInvalid" />
```

Reply:
BTW How cool is it that we now have multi line suggestions? 😄